covid_graphs/covid_graphs/formula_test.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+import math
+
+import numpy as np
+import pytest
+
+from .country_report import CountryReport
+from .formula import AtgFormula
+
+
+def test_two_curves():
+    cumulative_active = np.array([0, 0, 1, 2, 5, 18, 45])
+    start_date = datetime.date(2020, 4, 17)
+    dates = [start_date + datetime.timedelta(days=d) for d in range(len(cumulative_active))]
+    report = CountryReport(
+        short_name="UK",
+        long_name="United Kingdom",
+        dates=dates,
+        daily_positive=None,
+        daily_dead=None,
+        daily_recovered=None,
+        daily_active=None,
+        cumulative_active=cumulative_active,
+    )
+
+    f1 = AtgFormula(tg=2, a=47, exponent=1.5, min_case_count=2)
+    # The curve starts (t=1) at 2020-04-20. The maximum of this curve is at t = TG * exponent = 3.
+    max_t1, start_date1 = 3, datetime.date(2020, 4, 20)
+
+    f2 = AtgFormula(tg=3, a=12, exponent=2, min_case_count=1)
+    # The curve starts (t=1) at 2020-04-19. The maximum of this curve is at t = TG * exponent = 6.
+    max_t2, start_date2, length2 = 6, datetime.date(2020, 4, 19), math.ceil(3 * (2 + math.sqrt(2)))
+
+    curve1 = f1.get_curve(report)
+    assert pytest.approx(curve1.y_max, (47 / 2) * (max_t1 / 2) ** 1.5 * math.exp(-max_t1 / 2))
```

Comment:
This is not how `pytest.approx` works! :) See my original suggestion, we have to make a bugfix PR.
https://docs.pytest.org/en/latest/reference.html#pytest-approx

Reply:
Hahahahaha, I love how my brain just assumed that it knows the only possible API :D Fixed in 41ce34aafa939d84e9e10c2fdb82fb35f3c78f86